src/vuestic-theme/vuestic-components/va-navbar/VaNavbar.vue

```diff
@@ -116,7 +114,6 @@ $nav-border-side-width: 3.1875rem;
     height: 1rem;
     margin: auto;
     z-index: 2;
```

Comment:
We most probably don't need this line anymore as well. I previously fixed the same bug with this line, but your fix seems to be more robust.